Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1798,14 +1800,347 @@ def remove_nulls_from_dictionary(dict):
+
+        >>> build_dbot_entry('user@example.com', 'Email', 'Vendor', 3, 'Malicious email')
+        {'DBotScore': {'Indicator': 'user@example.com', 'Type': 'email', 'Vendor': 'Vendor', 'Score': 3},\
+ 'Account.Email(val.Address && val.Address == obj.Address)': {'Address': 'user@example.com', 'Malicious': \
+{'Vendor': 3, 'Description': 'Malicious email'}}}
+
+    :type indicator: ``str``
+    :param indicator: indicator field. if using file hashes, can be dict
+
+    :type indicator_type: ``str``
+    :param indicator_type:
+        type of indicator ('url, 'domain', 'ip', 'cve', 'email', 'md5', 'sha1', 'sha256', 'crc32', 'sha512', 'ctph')
+
+    :type score: ``int``
+    :param score: score (0, 1, 2 , 3)
+
+    :type vendor: ``str``
+    :param vendor: Integration ID
+
+    :type score: ``int``
+    :param score: DBot score (0-3)
+
+    :type description: ``str`` or ``None``
+    :param description: description (will be added to malicious if dbot_score is 3). can be None
+
+    :type build_malicious: ``bool``
+    :param build_malicious: if True, will add a malicious entry
+
+    :return: dbot entry
+    :rtype: ``dict``
+    """
+    indicator_type_lower = indicator_type.lower()
+    dbot_entry = {
+        outputPaths['dbotscore']: {
+            'Indicator': indicator,
+            'Type': indicator_type_lower,
+            'Vendor': vendor,
+            'Score': score
+        }}
+    if score is 3 and build_malicious:
+        dbot_entry.update(build_malicious_dbot_entry(indicator, indicator_type, score, description))
+    return dbot_entry
+
+
+def build_malicious_dbot_entry(indicator, indicator_type, vendor, description=None):
+    """ Build Malicious dbot entry
+    Examples:
+        >>> build_malicious_dbot_entry('8.8.8.8', 'ip', 'Vendor', 'Google DNS')
+        {'IP(val.Address && val.Address == obj.Address)': {'Address': '8.8.8.8', 'Malicious': {'Vendor': 'Vendor'\
+, 'Description': 'Google DNS'}}}
+
+        >>> build_malicious_dbot_entry('md5hash', 'MD5', 'Vendor', 'Malicious File')
+        {'File(val.MD5 && val.MD5 == obj.MD5 || val.SHA1 && val.SHA1 == obj.SHA1 ||\
+val.SHA256 && val.SHA256 == obj.SHA256 || val.SHA512 && val.SHA512 == obj.SHA512 || val.CRC32 && val.CRC32 ==\
+obj.CRC32 || val.CTPH && val.CTPH == obj.CTPH)': {'MD5': 'md5hash', 'Malicious': {'Vendor': 'Vendor',\
+'Description': 'Malicious File'}}}
+
+    :type indicator: ``str``
+    :param indicator: Value (e.g. 8.8.8.8)
+
+    :type indicator_type: ``str``
+    :param indicator_type: e.g. 'IP'
+
+    :type vendor: ``str``
+    :param vendor: Integration ID
+
+    :type description: ``str``
+    :param description: Why it's malicious
+
+    :return: A malicious DBot entry
+    :rtype: ``dict``
+    """
+    file_types = ('md5', 'sha1', 'sha256', 'crc32', 'sha512', 'ctph')
+    indicator_type_lower = indicator_type.lower()
+    if indicator_type_lower in ('ip', 'email'):
```

Comment:
add those checks to the build_dbot_entry function

Reply:
done